tests/output/ConsoleInteractionTest.py

```diff
@@ -507,13 +501,20 @@ def test_print_results_for_file(self):
                 {},
                 color=False)
             self.assertEqual("""\nfilename
-|   5| line•5
+|   5| {0}
 |    | [NORMAL] SpaceConsistencyBear:
-|    | Trailing whitespace found
-""",
-                             stdout.getvalue())
+|    | {1}
+""".format(highlightoutput('line 5'),
+                highlight("Trailing whitespace found", TextLexer(),
+                          TerminalTrueColorFormatter(style=BackgroundMessage
+                                                     )).rstrip("\r\n")),
+                stdout.getvalue())
 
     def test_print_results_sorting(self):
+        def highlightoutput(text):
+            return highlight(text, self.lexer, TerminalTrueColorFormatter(
+                style=BackgroundSourceRange)).rstrip("\n")
```

Comment:
move that function to class level, then you can remove all other declarations :)